addons/interactions/src/index.ts

```diff
@@ -0,0 +1,6 @@
+if (module && module.hot && module.hot.decline) {
+  module.hot.decline();
+}
```

Comment:
No idea, this comes from the addon kit. @winkerVSbecks ?

Reply:
@tmeasday  Addon kit was based on addons in the monorepo. They all seemed to use this. I wasn't sure if this was something that helped refresh addons during development. Should I get rid of it?

Reply:
I'm not really sure. What it will do is force a hard refresh if an addon changes, rather than allowing the preview.js file to pick it up and HMR.

Currently in 6.3 the preview always hard-refreshes anyway. 

I believe prior to 6.2 we had issues with decorators from addons re-applying when the preview HMR-ed, which I would guess was the reason for this code.

In 6.4, I am pretty sure the issue should be gone and hard-refreshes shouldn't be necessary, so, yeah TLDR, I think let's get rid of it, and see what happens at least. @ghengeveld perhaps can try it in this addon and see if there are negative side effects in development.

Reply:
I thought this code was an optimization wince we never HMR the manager anyway

Reply:
Oh is this manager side code? I have no idea then.

Reply:
Dropped it.